.travis.yml

```diff
@@ -19,6 +19,7 @@ before_install:
     - source ~/.bashrc
     - sh -e /etc/init.d/xvfb start
     - sleep 3
+script: make test
```

Comment:
needed because our root package.json now has a test script and travis use it by default

Reply:
Running Windows10, powershell. Installation is bumpy.
Need to install lerna & babel (easy...)
Still fighting the error: "Module not found: Can't resolve 'data-generator' in 'C:\dev\React\react-admin\examples\demo\src\fakeServer'"

Any idea?